src/emitter.rs

```diff
@@ -425,7 +428,7 @@ products:
   "true": bool key
   "{}": empty hash key
 x: test
-"y": "can't avoid quoting here"
+y: "can't avoid quoting here"
```

Comment:
it still says "can't avoid quoting here", which contradicts this change

Reply:
if it's the different treatment of say `"y": ...` and `x: ...`, why don't we just quote all keys in the export format? it's still valid yaml